src/ReferenceAnalyzer.Core/ReferenceAnalyzer.cs

```diff
@@ -33,10 +35,16 @@ public class ReferenceAnalyzer : IReferenceAnalyzer
         public bool ThrowOnCompilationFailures { get; set; } = true;
         public IProgress<double> ProgressReporter { get; set; } = new Progress<double>();
 
-        public async IAsyncEnumerable<ReferencesReport> AnalyzeAll(string solutionPath)
+        public IAsyncEnumerable<ReferencesReport> AnalyzeAll(string solutionPath)
```

Comment:
Why AnalyzeAll without CancellationToken is not async?

Reply:
I'd have to wrap it's overload call in `await foreach` and `yield` to make it async method. Not worth it IMO.